index.html

```diff
@@ -40,6 +40,27 @@
             </ul>
           </li>
         </ul>
+
+        <!-- mobile nav display -->
+        <div class="container-nav">
+          <ul class="mobile-nav-item">
+            <li class="mobile-nav-li">
+              <a href="#projects" class="mobile-nav-links">Portfolio</a>
+            </li>
+            <li class="mobile-nav-li">
+              <a href="#about-section" class="mobile-nav-links">About</a>
+            </li>
+            <li class="mobile-nav-li">
+              <a href="#contact-me" class="mobile-nav-links">Contact</a>
+            </li>
+          </ul>
+
+          <ul class="hamb-ul">
+            <li>
+              <a href="" class="close-hamb"> &times;</a>
+            </li>
+          </ul>
+        </div>
       </nav>
     </header>
     <main>
```

Comment:
Your mobile menu design doesn't quite match the template design.

| Your design  | Figma design |
| ------------- | ------------- |
| <img width="187" alt="Screen Shot 2022-06-13 at 11 34 40" src="https://user-images.githubusercontent.com/82426682/173401838-cfda1fa2-db2e-43da-8068-a0936218c99e.png">| <img width="207" alt="Screen Shot 2022-06-13 at 11 35 36" src="https://user-images.githubusercontent.com/82426682/173402002-08a7713d-f3a6-4823-88c7-ff1c52d3d8ab.png"> |

Please fix margins, padding, opacity, and the `X` icon in order to better match the template design.

<img width="844" alt="Screen Shot 2022-06-13 at 11 37 06" src="https://user-images.githubusercontent.com/82426682/173402243-2d9cb03f-9be0-4a7b-a80d-05af605d9131.png">